### PR TITLE
Correct docs for biganimal show-cluster-architectures

### PR DESCRIPTION
**Pull request by ftouserkani-edb**

### CODE REVIEW

File: product_docs/docs/biganimal/release/reference/cli.mdx [hunk 1]
```diff
@@ -258,7 +258,7 @@ biganimal create-cluster --clusterConfigFile ./config_file.yaml
 To query an enumeration of valid values for above BigAnimal and cloud service provider related properties, CLI provides you a series of subcommands, for example: you can use `show-cluster-architecture` to list all available cloud architecture to your current login account.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/reference/cli.mdx:258]:
```suggestion
To query an enumeration of valid values for above BigAnimal and cloud service provider related properties, CLI provides you a series of subcommands, for example: you can use `show-cluster-architectures` to list all available cloud architecture to your current login account.
```

**Comment by drothery-edb** [comment]
Added a few more reviewers. If one of you on the CLI dev team can review this PR including my suggested changes, we'll go ahead and publish the changes.

**Review by ftouserkani-edb — COMMENTED** [review]
It should read something like this as these are NOT features supplied by Cloud Service Providers. These available features are delivered with BigAnimal product in a particular Cloud service provider account.




The paragraph should read:
CLI provides a series of subcommands to query an enumeration of BigAnimal product  related properties. For example: you can use `show-cluster-architectures` to list all BigAnimal database architectures available within your Cloud Service Provider account.

**Comment by drothery-edb** [comment]
Changes made in https://github.com/EnterpriseDB/docs/pull/4680

**Pull request closed without merging**